src/components/map/map.tsx

```diff
@@ -226,6 +244,24 @@ export class Map implements GxComponent {
     }
   }
 
+  private onMapLineDeleted(line: polyline) {
+    //const i = 0;
```

Comment:
As a convention, we avoid as much as possible to commit commented code.

Reply:
Finally I decided to implement the linesList and the full method OnMapLineDeleted.
Addressed in b9c59f7